arcgis-ios-sdk-samples/AppDelegate.swift

```diff
@@ -169,9 +169,38 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
 }
 
 extension UIColor {
-    class var primaryBlue: UIColor { return #colorLiteral(red: 0, green: 0.475, blue: 0.757, alpha: 1) }
-    class var secondaryBlue: UIColor { return #colorLiteral(red: 0, green: 0.368, blue: 0.584, alpha: 1) }
-    class var backgroundGray: UIColor { return #colorLiteral(red: 0.973, green: 0.973, blue: 0.973, alpha: 1) }
-    class var primaryTextColor: UIColor { return #colorLiteral(red: 0.196, green: 0.196, blue: 0.196, alpha: 1) }
-    class var secondaryTextColor: UIColor { return #colorLiteral(red: 0.349, green: 0.349, blue: 0.349, alpha: 1) }
+    static let primaryBlue: UIColor = { return #colorLiteral(red: 0, green: 0.475, blue: 0.757, alpha: 1) }()
+    static let secondaryBlue: UIColor = { return #colorLiteral(red: 0, green: 0.368, blue: 0.584, alpha: 1) }()
+    
+    static let systemBackgroundWithOpacity: UIColor = {
+        if #available(iOS 13, *) {
+            return UIColor.systemBackground.withAlphaComponent(0.6)
+        } else {
+            return #colorLiteral(red: 1, green: 1, blue: 1, alpha: 0.6)
+        }
+    }()
+    
```

Comment:
I'm not pretty sure about the changes here. I would like to make them `IBInspectable`, but using Color assets makes sense as well: https://nshipster.com/dark-mode/#manage-colors-with-an-asset-catalog.

Should I instead create these colors as asset catalog to avoid future re-visits?
Using asset catalog would also be beneficial for changing theme colors.

Reply:
Update: this change will get reverted and created as a color asset catalog.